packages/taiko-client/pkg/rpc/client.go

```diff
@@ -62,19 +65,38 @@ func NewClient(ctx context.Context, cfg *ClientConfig) (*Client, error) {
 		l1BeaconClient *BeaconClient
 		l2CheckPoint   *EthClient
 		err            error
+		mevPool        *MevPool
```

Comment:
maybe rename "mevPool" to "privateTxPool"

Reply:
OK